openssl-dynamic/src/main/c/jnilib.c

```diff
@@ -411,7 +411,6 @@ jint JNI_OnLoad_netty_tcnative(JavaVM* vm, void* reserved) {
         return JNI_ERR;
     }
 #else
-    (void)parsePackagePrefix;
```

Comment:
thanks for the explanation @carl-mastrangelo ... please check the updated code which should address these issues. Let me know if this approach works for you.